src/libopensc/card-tcos.c

```diff
@@ -351,28 +351,28 @@ static int tcos_select_file(sc_card_t *card,
 	pathlen = in_path->len;
 
 	sc_format_apdu(card, &apdu, SC_APDU_CASE_4_SHORT, 0xA4, 0, 0x04);
-	
+
 	switch (in_path->type) {
-	case SC_PATH_TYPE_FILE_ID:
-		if (pathlen != 2) return SC_ERROR_INVALID_ARGUMENTS;
-		/* fall through */
-	case SC_PATH_TYPE_FROM_CURRENT:
-		apdu.p1 = 9;
-		break;
-	case SC_PATH_TYPE_DF_NAME:
-		apdu.p1 = 4;
-		break;
-	case SC_PATH_TYPE_PATH:
-		apdu.p1 = 8;
-		if (pathlen >= 2 && memcmp(path, "\x3F\x00", 2) == 0) path += 2, pathlen -= 2;
-		if (pathlen == 0) apdu.p1 = 0;
-		break;
-	case SC_PATH_TYPE_PARENT:
-		apdu.p1 = 3;
-		pathlen = 0;
-		break;
-	default:
-		SC_FUNC_RETURN(ctx, SC_LOG_DEBUG_VERBOSE, SC_ERROR_INVALID_ARGUMENTS);
+		case SC_PATH_TYPE_FILE_ID:
+			if (pathlen != 2) return SC_ERROR_INVALID_ARGUMENTS;
+			/* fall through */
+		case SC_PATH_TYPE_FROM_CURRENT:
+			apdu.p1 = 9;
+			break;
+		case SC_PATH_TYPE_DF_NAME:
+			apdu.p1 = 4;
+			break;
+		case SC_PATH_TYPE_PATH:
+			apdu.p1 = 8;
+			if (pathlen >= 2 && memcmp(path, "\x3F\x00", 2) == 0) path += 2, pathlen -= 2;
+			if (pathlen == 0) apdu.p1 = 0;
+			break;
+		case SC_PATH_TYPE_PARENT:
+			apdu.p1 = 3;
+			pathlen = 0;
+			break;
+		default:
+			SC_FUNC_RETURN(ctx, SC_LOG_DEBUG_VERBOSE, SC_ERROR_INVALID_ARGUMENTS);
```

Comment:
The same as above -- the switch and case should be on the same indentation level according to kernel coding style:

https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/Documentation/process/coding-style.rst#n47